jobs/JJOB_DA_PREP

```diff
@@ -10,14 +10,23 @@ module use -a /scratch2/NCEPDEV/marine/marineda/modulefiles/
 module load jedi-intel-17.0.5.239
 module load nco
 
+echo  
+echo "The system runs at $RUNCDATE"
+echo
+echo "Root GODAS " ${ROOT_GODAS_DIR}
+echo
 echo "SOCA_STATIC="$SOCA_STATIC
-echo $RUNCDATE
 
 mkdir -p $RUNCDATE
+
 cd $RUNCDATE
 
 # TODO: Hard-coded stuff that needs to be moved/changed
-export MODEL_SCRATCH=/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/mom6-cice5-fv3/test-initcice5
+
+export MODEL_SCRATCH=/scratch4/NCEPDEV/ocean/scrub/Guillaume.Vernieres/JEDI/mom6-cice5-fv3/test-initcice5/
+export case='en' #options 3dvar or en
```

Comment:
case needs to be passed as an argument/envvar ... not hard coded in this script.